core-bundle/src/Resources/contao/dca/tl_module.php

```diff
@@ -278,6 +278,12 @@
 			'inputType'               => 'checkbox',
 			'sql'                     => "char(1) NOT NULL default ''"
 		),
+		'trustedDevicesEnabled' => array
```

Comment:
I would suggest to remove this for now and maybe add a feature to configure it per root page or bundle config, similar to the enforced 2-factor.

Reply:
Fixed in c25176a8f.